_episodes/47.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: episode
+title: "47: Meet the new Trino maintainers"
+date: 2023-03-15
+tags: database memory in-memory apache ignite
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Trino releases"
+  desc: "411-416"
+  time: 
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Information Engineering at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [James Petty](https://github.com/pettyjamesm), ___ at AWS
```

Comment:
@pettyjamesm can you fill in what title you'd want here?

Reply:
“Senior Software Engineer” works